content/documentation/overview/index.adoc

```diff
@@ -37,7 +37,7 @@ icon:lightbulb[size=2x] {nbsp}{nbsp}{nbsp}{nbsp} Kiali also includes Jaeger Trac
 
 === Graph View
 
-Kiali provide a graph view of your namespace in real time, with the number of services and edges, we can see an overview with the inbound/outbound traffic the last minute.
+Kiali provides an interactive graph view of your namespace in real time, being able to display the interactions at several levels (applications, versions, workloads), with contextual information and charts on the selected node or edge.
```

Comment:
node is a confusing term :)
Because [Node in k8s](https://kubernetes.io/docs/concepts/architecture/nodes/) has a very specific meaning.
So perhaps a "graph node" would be better.